### PR TITLE
Rebase rte branch on top of master

### PR DESCRIPTION
**Pull request by cynepco3hahue**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -9,58 +9,51 @@ require (
...
+	k8s.io/component-base => k8s.io/component-base v0.19.4
+	k8s.io/cri-api => k8s.io/cri-api v0.19.4
+	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.19.4
+	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.19.4
+	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.19.4
+	k8s.io/kube-proxy => k8s.io/kube-proxy v0.19.4
+	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.19.4
+	k8s.io/kubectl => k8s.io/kubectl v0.19.4
+	k8s.io/kubelet => k8s.io/kubelet v0.19.4
+	k8s.io/kubernetes => k8s.io/kubernetes v1.19.4
```
Comment (swatisehgal) [inline go.mod]: We would still need to replace kubernetes 1.19.4 by a customized version as the kubernetes 1.19.4 does not have the podresource API changes we need (GetAvailable/GetAllocatable and cpuid and device topology information)  e.g.
`k8s.io/kubernetes => github.com/swatisehgal/kubernetes v1.19.0-beta.2.0.20200928123437-1c4f5ad40262` in order to be able to access the API.

**Review by swatisehgal — CHANGES_REQUESTED** [review]

File: go.mod (hunk 1 above)
Comment (cynepco3hahue) [reply go.mod]: true, missed it during the rebase

Comment (cynepco3hahue) [reply go.mod]: done

**Comment by swatisehgal** [comment]
Hi, the code is failing to build with the following error:

> undefined: grpc.SupportPackageIsVersion7
> undefined: grpc.ServiceRegistrar

Seems like the issue described in the links below: 
https://github.com/grpc/grpc-go#compiling-error-undefined-grpcsupportpackageisversion
https://stackoverflow.com/questions/63662787/undefined-grpc-supportpackageisversion7-grpc-serviceregistrar

Try running `make build` or `make image` to verify that the code builds.

I had faced the same issue and the following helped:
```
go get -u github.com/golang/protobuf/{proto,protoc-gen-go}
go get -u google.golang.org/grpc
```
To generate the code from proto file I used the following command:
`protoc --go_out=. --go_opt=paths=source_relative    --go-grpc_out=. --go-grpc_opt=paths=source_relative,require_unimplemented_servers=false  pkg/topologyupdater/topology-updater.proto`

**Comment by cynepco3hahue** [comment]
> Hi, the code is failing to build with the following error:
> 
> > undefined: grpc.SupportPackageIsVersion7
> > undefined: grpc.ServiceRegistrar
> 
> Seems like the issue described in the links below:
> https://github.com/grpc/grpc-go#compiling-error-undefined-grpcsupportpackageisversion
> https://stackoverflow.com/questions/63662787/undefined-grpc-supportpackageisversion7-grpc-serviceregistrar
> 
> Try running `make build` or `make image` to verify that the code builds.
> 
> I had faced the same issue previously had the following helped:
> 
> ```
> go get -u github.com/golang/protobuf/{proto,protoc-gen-go}
> go get -u google.golang.org/grpc
> ```
> 
> To generate the code from proto file I used the following command:
> `protoc --go_out=. --go_opt=paths=source_relative --go-grpc_out=. --go-grpc_opt=paths=source_relative,require_unimplemented_servers=false pkg/topologyupdater/topology-updater.proto`

It should be fixed by https://github.com/swatisehgal/node-feature-discovery/pull/2/commits/e3f4f75f9176f87fa1a38184e3efa8dc20185740

**Comment by cynepco3hahue** [comment]
Once this one will be merged I will rebase the https://github.com/swatisehgal/node-feature-discovery/pull/2

**Comment by swatisehgal** [comment]
LGTM! Approved. Thanks for your contribution!

**Pull request merged**